MapleServer2/PacketHandlers/Login/CharacterManagementHandler.cs

```diff
@@ -61,8 +61,8 @@ public void HandleCreate(LoginSession session, PacketReader packet)
         {
             byte gender = packet.ReadByte();
             //packet.ReadShort(); // const?
-            var jobCode = (Job)packet.ReadShort();
-            //int jobCode = packet.ReadShort();
+            // var jobCode = (Job)packet.ReadShort();
```

Comment:
is there a reason for keeping the commented out code?

Reply:
Since I don't know the origins of why the previous code was created I left it commented out.